jso/apis/src/main/java/org/teavm/jso/dom/events/WheelEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2014 Alexey Andreev.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.dom.events;
+
+import org.teavm.jso.JSProperty;
+
+/**
```

Comment:
Empty comment. Remove it or place something useful (like `@author` tag or documentation). Currently, I try to avoid `@author` comments at all, if I have nothing to write.